examples/src/indexes/vector_stores/hana_vector/internalEmbeddings.ts

```diff
@@ -0,0 +1,81 @@
+import { Document } from "@langchain/core/documents";
+import hanaClient from "hdb";
+import { HanaInternalEmbeddings } from "@langchain/community/embeddings/hana_internal";
+import { HanaDB, HanaDBArgs } from "@langchain/community/vectorstores/hanavector";
+
+// Initialize the internal embeddings instance using the internal model ID.
+// This instance will use SAP HANA's built-in VECTOR_EMBEDDING function of HanaDB.
+const internalEmbeddings = new HanaInternalEmbeddings({
+  internalEmbeddingModelId: process.env.HANA_DB_EMBEDDING_MODEL_ID || "your_model_id",
```

Comment:
```suggestion
  internalEmbeddingModelId: process.env.HANA_DB_EMBEDDING_MODEL_ID || "SAP_NEB.20240715",
```
I suggest to use an existing model Is as default value.

Reply:
Nice idea! Do you think we should also set a default value for the actual `HanaInternalEmbeddings` implementation?